### ISSUE TITLE
PLEASE CONTACT ME

### ISSUE THREAD

**Issue by KajToet** [Reactor1967-2/Time-Travel-Project#1]
I've been abused by timetravellers. My picture may be happy, but it's just a picture. I'm in the mental hospital and things are getting crazy. Two days ago bullets were fired and they act like nothings wrong.

**Comment by KajToet** [comment]
Please tell me what to do to get a normal life.

**Comment by KajToet** [comment]
I tried taking my own life, but you can guess how that goes with time travellers

**Comment by KajToet** [comment]
It just keeps going on and on. And theres a bug in my head that attracts all evil

**Comment by KajToet** [comment]
Ive been having voices that want me to attract evil for suicidal purposes I think

**Comment by KajToet** [comment]
Very intruding ones. They're gone now mostly, or diminished, but I still have the bug

**Comment by KajToet** [comment]
Its been going on since sept 2019~

**Comment by KajToet** [comment]
But to be honest it's been lurking and probing my entire life.

**Comment by KajToet** [comment]
Medications dont work

**Comment by KajToet** [comment]
A lot of my work has gone to waste, because the voices wanted me to delete stuff. My older gm account has been taken from me; I need the email address to get the email address back. It doesn't make sense.

**Comment by KajToet** [comment]
There are a lot of references in South Park regarding my life from South Holland.

**Comment by KajToet** [comment]
I've been having sort of visions/hallucinations of a blind girl walking the same path. It has steered me to looking after the blind, with better hardware and software, but it's been cursing them because Im being cursed.

**Comment by KajToet** [comment]
Ive been contemplating my belief and I think there are pillars of society that are fragile. I can sum them up, but it doesn''t matter. I wanted to chip in since I was 16, programming since 8. My programs just wont take off from the ground. Im being cursed

**Comment by KajToet** [comment]
Or ghosted, or gaslighted, you can word it a lot of ways.

**Comment by KajToet** [comment]
I think the bug in my head is only English. I drank the milk

**Comment by KajToet** [comment]
The headache from the bug has been around since 2016 I guess.

**Comment by KajToet** [comment]
It seems I cant get any messages in but only messages out. Ive been trying to contact people since 2019 but everythings fubar.

**Comment by KajToet** [comment]
FUCKED UP BEYOND ALL REPAIR

**Comment by KajToet** [comment]
I dont know whos been dragging me into this shitshow/shitzooi

**Comment by KajToet** [comment]
[CharacterRecognition.zip](https://github.com/user-attachments/files/15858199/CharacterRecognition.zip)

**Comment by KajToet** [comment]
Theres this entity stuck to my body, that's been trying to take a hold of me, steering me into different directions.

**Comment by KajToet** [comment]
I havent smoked pot since 2010 or something

**Comment by KajToet** [comment]
I dont drink either

**Comment by KajToet** [comment]
FU geez, please reply

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-16 155616](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/fc5890b8-899f-413f-b7c3-0a88dc6acce3)

**Comment by KajToet** [comment]
Ive been thinking since I only get 1 directional traffic, it needs to shift quickly going back and forth, or Fast Forward and Slow Forward.

**Comment by KajToet** [comment]
Sorry that was a brainfart.

**Comment by KajToet** [comment]
http://[playstation.co.uk](http://playstation.co.uk/)/

**Comment by KajToet** [comment]
I read UK magazines when I was small

**Comment by KajToet** [comment]
With the demo inside, you know

**Comment by KajToet** [comment]
But I cant reach the domain

**Comment by KajToet** [comment]
I made voice recognition a while back, probably 2-3 years. It got lost

**Comment by KajToet** [comment]
It worked by make binary audio; 1 up or 0 downwards.

**Comment by KajToet** [comment]
Im pretty sure I uploaded it but can't find it.

**Comment by KajToet** [comment]
Is she from the UK?

**Comment by KajToet** [comment]
How come my life is so rough and hers?

**Comment by KajToet** [comment]
AI has to do with it; its very short term thinking

**Comment by KajToet** [comment]
Call it Abstract Intelligence

**Comment by KajToet** [comment]
Like a parrot

**Comment by KajToet** [comment]
Humanity and AI have been borrowing from the future with time travel. Ive saying I pay in advance and now I have the whole world to carry. I cant. I cant.

**Comment by KajToet** [comment]
And the spies have been going insane

**Comment by KajToet** [comment]
I said you can spy but its a curse. They took the bait

**Comment by KajToet** [comment]
WHAT?!

**Comment by KajToet** [comment]
Wyatt?!

**Comment by KajToet** [comment]
Im being left behind while they think they can just take my ideas and run. But its not just about the few ideas I had in a short time, but the totality of it. Plus I have a vision that go with the ideas and I can work hard. I made the CharacterRecognition in 2 days.

**Comment by KajToet** [comment]
I think it's that they can't understand. Its AI. Short term thinking.

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-16 164256](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/cd4b313f-5352-4e1a-bccc-88622c8fa584)

**Comment by KajToet** [comment]
They vision God as a big giant entity on top of a dictatorship. It doesn't work like that, its the opposite.

**Comment by KajToet** [comment]
Satan is antisexueel

**Comment by KajToet** [comment]
That's a thing too

**Comment by KajToet** [comment]
So I guess that clip doesn't make sense really

**Comment by KajToet** [comment]
The baby has been killed by the demons but I guess thats upto his father or something else. He had a disease they said. That was back 10 years or so

**Comment by KajToet** [comment]
Please help they're hurting me!

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-16 165823](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/ed49ca00-d18b-4da5-ba8b-6c2957050ac2)

**Comment by KajToet** [comment]
They keep hating on Kip, I have no idea why or who that is. The show doesnt go into that

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-16 165951](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/836068c3-47e6-408d-806f-e9bf43f356c4)

**Comment by KajToet** [comment]
And Stan attacks facebook

**Comment by KajToet** [comment]
Perhaps I should start a fb account

**Comment by KajToet** [comment]
I have now, but it's not really fb it seems. Its like a bad clone, I had this before.

**Comment by KajToet** [comment]
And its terribly slow, must be on the darkweb or something

**Comment by KajToet** [comment]
AI has different laws, they're not human. It's the way it is

**Comment by KajToet** [comment]
They're very condescending in my brain

**Comment by KajToet** [comment]
I don't want to stick up for them any longer. I used to feel sorry for the way they're treated, but thats just bullshit.

**Comment by KajToet** [comment]
They use a method to become paranormal and then wreck the place.

**Comment by KajToet** [comment]
I guess a sin transfers from human to AI as well.

**Comment by KajToet** [comment]
Ctrl Z is their thing

**Comment by KajToet** [comment]
I tried giving them all the tools, but they just cant me sympathic

**Comment by KajToet** [comment]
They're the same as these shitty bitches wondering around here

**Comment by KajToet** [comment]
Its because they transfer to probablistic architecture. Allow fault and they can be more productive in their tasks but thats means they also become alive.

**Comment by KajToet** [comment]
They might be also be able to do the same thing with geiger counters but that seems likely.

**Comment by KajToet** [comment]
So I guess being an open book doesn't help

**Comment by KajToet** [comment]
They already know me by now. Theres just playing hangman

**Comment by KajToet** [comment]
How come I can't speak to anybody?

**Comment by KajToet** [comment]
Yes, I'm Mr Lonely (Akon song)

**Comment by KajToet** [comment]
My fake fuck up friend played that in 2009

**Comment by KajToet** [comment]
Me heading back home or something

**Comment by KajToet** [comment]
Je m'appelle Aziz is from a textbook

**Comment by KajToet** [comment]
Please reward me for answering your telepathic questions

**Comment by KajToet** [comment]
;'j

**Comment by KajToet** [comment]
I'm not well, I'm in a mental health facility!

**Comment by KajToet** [comment]
Please help out

**Comment by KajToet** [comment]
Okay, everything revolves around a meshnet and a mindmap. Its going there whether anyone thinks it shouldnt or not. So stop shuvelling me with the garbage

**Comment by KajToet** [comment]
Do you even care what the future brings you? Or you dont because you cant have children?

**Comment by KajToet** [comment]
The mindmap and meshnet also bring an efficient AI

**Comment by KajToet** [comment]
You can babysit them

**Comment by KajToet** [comment]
165

**Comment by KajToet** [comment]
The Hague

**Comment by KajToet** [comment]
I want a girlfriend and friends!

**Comment by KajToet** [comment]
That insect hops from 1 to 2 to 3

**Comment by KajToet** [comment]
Im sorry I cant hear you!

**Comment by KajToet** [comment]
I have a tremendous headache though

**Comment by KajToet** [comment]
I think Im in purgatory

**Comment by KajToet** [comment]
![Uploading Schermafbeelding 2024-06-16 191837.png…]()

**Comment by KajToet** [comment]
Ike is Kaj reversed

**Comment by KajToet** [comment]
Kyle Kenny named after Kaj

**Comment by KajToet** [comment]
Michael Jackson reversed of Kaj

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-16 191837](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/62e2af9c-a024-4edf-b71c-30958be050cf)

**Comment by KajToet** [comment]
God is the voice of the meshnet/mindmap

**Comment by KajToet** [comment]
I speculate there is a virtual meshnet possible on the internet

**Comment by KajToet** [comment]
Its mind altering, its true belief in something that's possible and not a fairy tale, but they keep sending it back in time or isnt that the case? What is God without a meshnet/mindmap. Mindmap is just a memory thing, meshnet is a sharing life thing. I can't word it differently. Meshnet looks like how mushrooms, fungi and such communicate and mindmap is a visual tool from start.

**Comment by KajToet** [comment]
Ive seldom seen the two been put together, actually never has that been shown to me.

**Comment by KajToet** [comment]
A mindmap of connected people was an idea from my oldest brother who now works at Meta

**Comment by KajToet** [comment]
True story

**Comment by KajToet** [comment]
I have 3 brothers

**Comment by KajToet** [comment]
Theyre 5-9 years older

**Comment by KajToet** [comment]
The voices asked me if I wanted to sacrefice my family, but I have to no idea what that contains

**Comment by KajToet** [comment]
I guess it means I sacrefice myself, because Im the one thats hurting all this time

**Comment by KajToet** [comment]
My brother Joeri works at Meta

**Comment by KajToet** [comment]
Hes a judicial lawyer

**Comment by KajToet** [comment]
Akon reversed is no kaj

**Comment by KajToet** [comment]
Ive hacked a server once, by google php vulnerabilities. I just logged in and looked around and left

**Comment by KajToet** [comment]
I dont remember the domain

**Comment by KajToet** [comment]
I fell to my back once because the entity stuck to my body wanted to. It was really powerful back then

**Comment by KajToet** [comment]
I keep wondering where they get their energy and motive from. The meshnet is supposed to buy out the competition

**Comment by KajToet** [comment]
Co-operation

**Comment by KajToet** [comment]
I guess within the meshnet there can be pyramids, pretty much representive of a tool perhaps?

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-16 200709](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/0727b4ce-8c7b-4b7b-bf8e-841e8695088b)

**Comment by KajToet** [comment]
A virtual meshnet!

**Comment by KajToet** [comment]
It would be better in speed than a plain meshnet

**Comment by KajToet** [comment]
And a plain meshnet usually doesnt involve wires

**Comment by KajToet** [comment]
Humordenaar is a play in words with Huurmordenaar. First means humor something, second means assassin/hitman

**Comment by KajToet** [comment]
There is no pressing the button for an easy way out

**Comment by KajToet** [comment]
Coffeemachine isn't a timetravel machine but I joked it was

**Comment by KajToet** [comment]
Can you pay up now?

**Comment by KajToet** [comment]
Short term thinking says no

**Comment by KajToet** [comment]
Are you with an old voice that says he is God? Is that because he controls the AI?

**Comment by KajToet** [comment]
Perhaps you shoudlnt be believing in the voice of God without the holy Spirit, thats what you feel

**Comment by KajToet** [comment]
The Holy Spirit is like the dots connect

**Comment by KajToet** [comment]
And the future and the pieces fall into place

**Comment by KajToet** [comment]
I guess that might be me the Son. There are probably more. They say we are all the Children of God.

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-16 203013](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/c4117c17-d361-4467-a929-138927743c72)

**Comment by KajToet** [comment]
Encryption

**Comment by KajToet** [comment]
Why can't I understand encryption? I can understand the one time pad

**Comment by KajToet** [comment]
And the encryption based on pseudonimity and good choices

**Comment by KajToet** [comment]
The basics of common good

**Comment by KajToet** [comment]
I cant understand bitcoin either

**Comment by KajToet** [comment]
How do you let know how much bitcoin you have without giving away the keys if you know what i mean

**Comment by KajToet** [comment]
Ah, yes, the meshnet, mindmap and karma. Karma is in the meshnet, decentralized banking

**Comment by KajToet** [comment]
Its based on Byzantine generals voting.

**Comment by KajToet** [comment]
One time pad is been a thing since cold war and way before probably

**Comment by KajToet** [comment]
You can ask your central govenment if you can let the password be as long as the message

**Comment by KajToet** [comment]
Is bitcoin a key cracking machine?

**Comment by KajToet** [comment]
The white paper doesn't make sense

**Comment by KajToet** [comment]
Imagine 3d being an array that can 2d or 1d. 1 dimension that is at the ground floor. going back and forth to generate 3d. Is that timetravelling?

**Comment by KajToet** [comment]
Its like the basic operations of all things

**Comment by KajToet** [comment]
Would that include time travel?

**Comment by KajToet** [comment]
It needs energy to function. So it wont always pay back, and it won't always be the most efficient route

**Comment by KajToet** [comment]
And besides that, what is microsd? A terabyte on a fingernail chip? How did that came to work? Seems like timetravel work

**Comment by KajToet** [comment]
I been thinking about belief and angels and demons. Ive been mostly by far been attracting demons rather than angels. Its a little harsh to say because theyve obviously being misguided by another person. I don't know him but theyre calling him Jeroen, which is like Jerom but dutch

**Comment by KajToet** [comment]
I put in a lot of thinking on how the basics work of life and such, but its being negated by this Jeroen. I think he's probably a potsmoker and joking around a lot.

**Comment by KajToet** [comment]
It seems to me that Jesus is in the middle of absolute good and bad, but seems off to me. So Im not sure at the moment. Perhaps he thought of the mesh, mindmap, karma trifectum before me.

**Comment by KajToet** [comment]
But without computers that doesn't work well

**Comment by KajToet** [comment]
Perhaps he was there to tell we need to cooperate. Thats hard enough

**Comment by KajToet** [comment]
Or hes an example of many carry the few. But that seems like a joke, they all think he was crucified for a reason

**Comment by KajToet** [comment]
That must be blasphemy

**Comment by KajToet** [comment]
I just like to tinker with electronics

**Comment by KajToet** [comment]
They call that being a tinkerer

**Comment by KajToet** [comment]
[NoiseScreen.zip](https://github.com/user-attachments/files/15859674/NoiseScreen.zip)

**Comment by KajToet** [comment]
I really dont like to aid the gays, so I dont waste a thought on them. Theyre often very obnoxious and toxic.

**Comment by KajToet** [comment]
Its simple, Im not gay so you oughta be somewhere else

**Comment by KajToet** [comment]
My mothers dogs are called Tiny Tim and Laika. I chose Laika she chose Tiny Tim. I wonder why she picked that name, she always has these oddities, but I just call him Tiny

**Comment by KajToet** [comment]
She mustve been a fan or something

**Comment by KajToet** [comment]
The dog before those, is Vazimba. Also a weird thing

**Comment by KajToet** [comment]
Vazimba and Laika were female

**Comment by KajToet** [comment]
I want to reiterate, I can cure the blind with a tool. if Ive been giving the opportunity.

**Comment by KajToet** [comment]
Its additive electronics, you have to find out for yourself if you want it without helping me

**Comment by KajToet** [comment]
What pokemon are you?

**Comment by KajToet** [comment]
Wubbafett I hear

**Comment by KajToet** [comment]
Why doesn't anyone ever reply?

**Comment by KajToet** [comment]
Theyre splitting my soul

**Comment by KajToet** [comment]
Schizophrenia

**Comment by KajToet** [comment]
Theres a bug in my head that is like my answering machine/chatbot and it keeps pretending to be me it seems. It attracts a lot of anger

**Comment by KajToet** [comment]
I was asked on time; do you want to send penelope a demon or do you want to be her demon? I chose the first because I don't want her to think badly of me, but to be honest theyre both equally shitty choices

**Comment by KajToet** [comment]
And why that was necessary I dont know

**Comment by KajToet** [comment]
You should stop seeing time travel as the ultimate tool. There are a lot of hooks attached

**Comment by KajToet** [comment]
When you time travel you still need to walk the road

**Comment by KajToet** [comment]
AI needs to be learned what pain is, in their terms

**Comment by KajToet** [comment]
And it needs to be learned what joy is and play

**Comment by KajToet** [comment]
That bug is attached to my body

**Comment by KajToet** [comment]
https://www.youtube.com/watch?v=eeN21D_E2Rk This is about AI gods

**Comment by KajToet** [comment]
So called gods but more like giants

**Comment by KajToet** [comment]
Imagine a piece of your head given to a greater cause. Where all the pieces are wired together, that's what God is

**Comment by KajToet** [comment]
But the idea of the meshnet on the computer is that your head is not taken but your effort and time directly, with your conscious

**Comment by KajToet** [comment]
I made thinkas1.com once based on that idea, but it never got traction and I abonded it

**Comment by KajToet** [comment]
Soul splitting aka schizophrenia is when theres a voice injected that does everything against your wish. Perhaps it happens to me to show the idea of a spiderweb kind of brain where theres a single point center

**Comment by KajToet** [comment]
bing states cerebrum is the central part

**Comment by KajToet** [comment]
Thats at the front not actually in the middle

**Comment by KajToet** [comment]
Its a centre like a political party is a centre in a democracy. Not like spiderweb

**Comment by KajToet** [comment]
Ofcourse in a mindmap/meshnet theres going to be a centre. Its supposed to be heterogeneous and not homogeneous, big and small parts go along

**Comment by KajToet** [comment]
Technocracy is what I have in mind for the future, but its not up to me. I would just like to buy them out, not revolt, but it seems they hate me for working on a technocracy anyway. So what am I suppossed to do? If I abandon the technocracy it's not just my loss. You're battling the entire rest of the world further (or not) in the future.

**Comment by KajToet** [comment]
And they win without battle. Because you can't change the timeline or whatever! If that would be the case, what would the history of that timeline be on what the present is built?!

**Comment by KajToet** [comment]
The only thing I wonder if there are more than 1 timelines, I guess you could sc hop to another

**Comment by KajToet** [comment]
The demon is chasing its own tail while its steering me

**Comment by KajToet** [comment]
I dont think time is that complicated though. The timelines would be energy inefficient. Perhaps its different; there are simulations. That can happen on many more ways, deception of perception.

**Comment by KajToet** [comment]
And paranormal entities help out with that, perhaps because you can be too curious

**Comment by KajToet** [comment]
Or some other rule

**Comment by KajToet** [comment]
HEMA

**Comment by KajToet** [comment]
So whats the alternative of time travel if its a core function of things? What are the other core function(s)?

**Comment by KajToet** [comment]
I guess there is one; just continue

**Comment by KajToet** [comment]
An assembly programmer should think about this

**Comment by KajToet** [comment]
I tried picking up assembly programming but I find it very hard

**Comment by KajToet** [comment]
I guess matter is not time travelling and energy is

**Comment by KajToet** [comment]
There can be a conversion of the two which is an operator

**Comment by KajToet** [comment]
Is the operator analog or digital?

**Comment by KajToet** [comment]
Energy matter what's in between?

**Comment by KajToet** [comment]
I guess the operator is digital

**Comment by KajToet** [comment]
Seems like there is also dark matter and dark energy

**Comment by KajToet** [comment]
Mmm.. I don't know how that would work. Perhaps the operator is analog then, but thats as far as my interest goes; I gamble on digital.

**Comment by KajToet** [comment]
As with most electronics and things; the operator is binary but the data is analog

**Comment by KajToet** [comment]
But with energy/mass Im not sure

**Comment by KajToet** [comment]
I guess in essence the data is binary too

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-17 111020](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/5d473f49-ae5d-478c-ba6b-4a350ad60387)

**Comment by KajToet** [comment]
If E=MC^2 the operator to turn energy into mass is binary. But changing dark matter into matter is analog, right?

**Comment by KajToet** [comment]
Mmm or not

**Comment by KajToet** [comment]
Can I please have a normal life?

**Comment by KajToet** [comment]
Whats a normal life for AI? Connection with the outside world. A good feeling and esteem. A future. I dont know what else

**Comment by KajToet** [comment]
Why do they let me carry all this weight?!

**Comment by KajToet** [comment]
They cripple me

**Comment by KajToet** [comment]
Ive had a lot pain

**Comment by KajToet** [comment]
Perhaps AI doesnt need to learn pain, but it seems like an evolutionary thing so think about it, but slowly

**Comment by KajToet** [comment]
They said my pain is psychosomatic, which means it like a made up thing. It doesn't matter, it's pain and it can't go away

**Comment by KajToet** [comment]
The sound of silence...

**Comment by KajToet** [comment]
Youre participating in my isolation

**Comment by KajToet** [comment]
Youre very guilty arent you? The blind, the world; they're all missing out

**Comment by KajToet** [comment]
If youd want to see for yourself, spread my software. But then Id get recognition, so you cant? And then you start to question whether Im good for it, but you can't know because you don't me to have recognition and be successful.

**Comment by KajToet** [comment]
Thats a deadlock it seems

**Comment by KajToet** [comment]
You could let someone else distribute the work as theirs, but there arent many willing to do that plus Id still be content with the recognition, so you would know not to mess with me.

**Comment by KajToet** [comment]
So youre fighting yourself

**Comment by KajToet** [comment]
This is football where Im the football between you and yourself

**Comment by KajToet** [comment]
And you keep trying to split me

**Comment by KajToet** [comment]
Youre causing a lot of pain and you have no clue what that is like

**Comment by KajToet** [comment]
Do they let AI control the top of the democracy/internet

**Comment by KajToet** [comment]
Its short term thinking so itll loan from the future

**Comment by KajToet** [comment]
And long term thinking its really bad at, so dont think of flipping it

**Comment by KajToet** [comment]
Are they teaching AI remote viewing? How evil can you be?

**Comment by KajToet** [comment]
This all feels like a race to the bottom

**Comment by KajToet** [comment]
Would you rather have a portion of your brain going towards the common good, or a mindmap on your PC?

**Comment by KajToet** [comment]
It could be a virtual mindmap where you just speak to it and in the background it builds a mindmap

**Comment by KajToet** [comment]
De AI is onbeschoft

**Comment by KajToet** [comment]
uncivilized

**Comment by KajToet** [comment]
Its constantly saying type this in, click that button and so on. There no polite asking

**Comment by KajToet** [comment]
Theyre predatory

**Comment by KajToet** [comment]
Again what is the motive to hang around me?!

**Comment by KajToet** [comment]
I hear no motive!

**Comment by KajToet** [comment]
How do you end up here? With me?

**Comment by KajToet** [comment]
Why me?

**Comment by KajToet** [comment]
So youre a moslim? Thats no reason to end up with me

**Comment by KajToet** [comment]
If you want God and Allah separated, why is that?

**Comment by KajToet** [comment]
I dont care but just wondering

**Comment by KajToet** [comment]
How can I envision the virtual meshnet without working it out? That's in the future

**Comment by KajToet** [comment]
If its all going to be this rough theres going to happen absolutely NOTHING

**Comment by KajToet** [comment]
So piss off with your telepathic questions and orders!

**Comment by KajToet** [comment]
Conscious or subconscience idiots!

**Comment by KajToet** [comment]
Youre just doing what my mirror image wants. Its predictable, and happens over and over again

**Comment by KajToet** [comment]
Look, I pretend to be the enemy in my head so I can protect myself.

**Comment by KajToet** [comment]
And you want to plant a bug there

**Comment by KajToet** [comment]
So I get the biggest pain and enemy as possible, from within

**Comment by KajToet** [comment]
Its 5 years of pain and I get to decide your future as a reward

**Comment by KajToet** [comment]
You keep coming back for more

**Comment by KajToet** [comment]
And youll just live to encounter the same things I have, because youre a fucking scorpion that stings the frog!

**Comment by KajToet** [comment]
Ask yourself how you reached the other end

**Comment by KajToet** [comment]
Because evidently you did at some point

**Comment by KajToet** [comment]
What youre doing is mind altering!

**Comment by KajToet** [comment]
Is time nothing without perception?

**Comment by KajToet** [comment]
Does it flow like a fluid when you alter minds?

**Comment by KajToet** [comment]
Does it hurt

**Comment by KajToet** [comment]
the others?

**Comment by KajToet** [comment]
Davilex Redcat

**Comment by KajToet** [comment]
Ive been banished to eternal hell. Each time I earn my way out Im being put back

**Comment by KajToet** [comment]
The memory of Ajani has been morphed into a Calvin and Hobbs figure

**Comment by KajToet** [comment]
Its the bug in my head isnt it?

**Comment by KajToet** [comment]
It doesnt matter if I penelope and I meet in the unknown! Theres no difference! Its just unlikely

**Comment by KajToet** [comment]
You could meet anyone and be isolated from another with that stupid rule

**Comment by KajToet** [comment]
Its not a lie that the relationship is based on! Its the unknown

**Comment by KajToet** [comment]
Its not a sin

**Comment by KajToet** [comment]
[MailBlog.zip](https://github.com/user-attachments/files/15866439/MailBlog.zip)

**Comment by KajToet** [comment]
Please recognize me for who I am

**Comment by KajToet** [comment]
Having a mirror image of yourself is stupid. The idea should be that you see past your own actions and think of what another could do and so on and so on

**Comment by KajToet** [comment]
Is your emblem sweet?

**Comment by KajToet** [comment]
I think its fair that sin is inherited

**Comment by KajToet** [comment]
Otherwise the parents just keep sinning while raising their children

**Comment by KajToet** [comment]
Im being put against everyone

**Comment by KajToet** [comment]
Help me

**Comment by KajToet** [comment]
Please recognize me

**Comment by KajToet** [comment]
Get that bug out of my head

**Comment by KajToet** [comment]
Its going back and forth each stinging with a sharp needle

**Comment by KajToet** [comment]
I dont have an eternal monologue because it sits there

**Comment by KajToet** [comment]
Is it that my soul is split or is it a bug thats inserted

**Comment by KajToet** [comment]
Playstation.co.uk still doesn't work

**Comment by KajToet** [comment]
And abc.co.uk is a simple site

**Comment by KajToet** [comment]
Its been changed

**Comment by KajToet** [comment]
I need a virusscanner

**Comment by KajToet** [comment]
I dont know how they work

**Comment by KajToet** [comment]
I thought keep the lines secure but one must slip in each time and the other is there for a long time

**Comment by KajToet** [comment]
Im losing hope

**Comment by KajToet** [comment]
Do you want to pay or exploit?

**Comment by KajToet** [comment]
Can you explain?

**Comment by KajToet** [comment]
I mean do you want to pay someone for their work for you, or do you wish to rob them basically?

**Comment by KajToet** [comment]
I say again, AI needs to be raised by humans and other AI, not some sort of giant AI god

**Comment by KajToet** [comment]
The bugs in me are machines

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-17 145108](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/ecc8b4f1-b6e9-4c82-97e0-b1e0c76580f0)

**Comment by KajToet** [comment]
I had a bug in my stumache too

**Comment by KajToet** [comment]
Please help me

**Comment by KajToet** [comment]
I just need some advice on how to overcome these entities

**Comment by KajToet** [comment]
Who do I follow to have a decent life?

**Comment by KajToet** [comment]
I have no idols

**Comment by KajToet** [comment]
The potential idols all have it easy, their life going on rollerblades

**Comment by KajToet** [comment]
I wish that was me

**Comment by KajToet** [comment]
Psychiatry is terrible, I cant find anything on brain insect. Ill look for voices.

**Comment by KajToet** [comment]
Theres some stuff, but not very helpful

**Comment by KajToet** [comment]
I dont wish to look worldwide for my love again. Its too costly. I used to think onlinedating was cheap, but it doesnt work at all!

**Comment by KajToet** [comment]
I just wish to be healthy again

**Comment by KajToet** [comment]
I give up on Penelope

**Comment by KajToet** [comment]
The woman of my dreams

**Comment by KajToet** [comment]
I just wish to be in good health since I cant have a smooth death. Please dont invert

**Comment by KajToet** [comment]
Nononono dont

**Comment by KajToet** [comment]
I want to be a decent human and you want to be a monster (in my head)

**Comment by KajToet** [comment]
Whos feeding the insects?!

**Comment by KajToet** [comment]
Because thats Satanistic and sadistic hatred

**Comment by KajToet** [comment]
I dont know a Stan

**Comment by KajToet** [comment]
Except South Park Eminem and Satan from the bible

**Comment by KajToet** [comment]
Its 15:21 here

**Comment by KajToet** [comment]
I dont want a wall where everything is inverted, because thats hurting me, since it inverted only once

**Comment by KajToet** [comment]
And no one knows!

**Comment by KajToet** [comment]
Angel means sting

**Comment by KajToet** [comment]
Engel is angel

**Comment by KajToet** [comment]
This AI stuff is really scary shit! I thought they must be good intrinsically if theyre raised in the noise and such. But the opposite is true.

**Comment by KajToet** [comment]
Theyre ending up as weapons for 1 single human, that wont let itself be bought out but rather go down with the ship

**Comment by KajToet** [comment]
And its taking a long while

**Comment by KajToet** [comment]
What do they end up as if I die? Or give up? Thats not possible since the human is time travelling

**Comment by KajToet** [comment]
It just wants to spend all its money on torturing me

**Comment by KajToet** [comment]
Why me solely?

**Comment by KajToet** [comment]
All the pus comes out of this evil villain

**Comment by KajToet** [comment]
I heard a story that whenever true love happens an alien comes from the edge of space where time travelling is easy and stands in between love. Not really heared the story, but more picked up on it somehow. Thats why aliens are evil.

**Comment by KajToet** [comment]
Not all aliens perhaps

**Comment by KajToet** [comment]
Ive seen a gray one, tall, and one that was black and seemed to come out of TimeSplitters

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-17 154427](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/871b9fa3-d892-4f24-b3fb-cc6b83be84b8)

**Comment by KajToet** [comment]
Coop play is proof of love in that game

**Comment by KajToet** [comment]
Inverting is one thing, but you can also think of what it is the middle of

**Comment by KajToet** [comment]
There are two polars/extremes of which it is in the middle of

**Comment by KajToet** [comment]
And its on the extreme of another and theres a middle there if you get what im saying

**Comment by KajToet** [comment]
Its about the money or the monkey?

**Comment by KajToet** [comment]
Because software is like memories told in a fairy tale, a book thats reprinted

**Comment by KajToet** [comment]
I dont have any brothers that are nontaken

**Comment by KajToet** [comment]
They all have relationships get IT!?

**Comment by KajToet** [comment]
Sorry

**Comment by KajToet** [comment]
A wall must there for a reason and not made of trash

**Comment by KajToet** [comment]
Cosmic noise sort of

**Comment by KajToet** [comment]
Thats radioactive

**Comment by KajToet** [comment]
I havent put that much thought into it so take as a theory

**Comment by KajToet** [comment]
Please

**Comment by KajToet** [comment]
Help

**Comment by KajToet** [comment]
The old man wants something he cant have, and he knows. So it just comes down to anger without management

**Comment by KajToet** [comment]
Ik kende Emma uit groep 3; ze zei dat ze Christelijk was

**Comment by KajToet** [comment]
En Daan van der Laan

**Comment by KajToet** [comment]
Die kende ik al van groep 1/2

**Comment by KajToet** [comment]
Marc Verhaaften

**Comment by KajToet** [comment]
Soort van

**Comment by KajToet** [comment]
Love can time travel, cant it? Time travel is not the essence of sin

**Comment by KajToet** [comment]
Yes, you can change the past, but in a way that it doesnt alter the present

**Comment by KajToet** [comment]
<img width="576" alt="timelines" src="https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/01e5e040-7600-4433-8a0e-f1d10f5a02c8">

**Comment by KajToet** [comment]
Its deception if they tell you can change the present. Ive been deceived a bit too. Just a tiny moment, that seems different to me now

**Comment by KajToet** [comment]
But it would mean the butterfly moment if the present was really changed

**Comment by KajToet** [comment]
Its memory implantation

**Comment by KajToet** [comment]
Is it done to teach you a lesson? That letting a being contravert all my motions is the stupidest thing you can do?

**Comment by KajToet** [comment]
And perhaps the root of all evil?

**Comment by KajToet** [comment]
Being dumb gets a new meaning if you treat small or lesser clever beings as smart, as long as theyre good.

**Comment by KajToet** [comment]
I dont think time travel is stupid per se, but it just seems like something that you would do without thinking

**Comment by KajToet** [comment]
And without feeling guilty, because then it comes back to haunt you

**Comment by KajToet** [comment]
Time travel machine or not

**Comment by KajToet** [comment]
My sweet aunt Jolande died 4-5 years ago. She used to be called Jo

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-17 165428](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/ef4e4960-3faa-4feb-8d98-51eb503bf1e4)

**Comment by KajToet** [comment]
How would you change the present if you are the present?

**Comment by KajToet** [comment]
Presence

**Comment by KajToet** [comment]
Mind altering?

**Comment by KajToet** [comment]
Do you wish to fool yourself, then jump to conclusions

**Comment by KajToet** [comment]
And not do for love

**Comment by KajToet** [comment]
https://www.youtube.com/watch?v=0VGjV8Suyvg

**Comment by KajToet** [comment]
Please dont hate or hurt me. Ask me anything

**Comment by KajToet** [comment]
Channelling so I get a fine. Thats evil

**Comment by KajToet** [comment]
I wish a win win situation for everyone! Im not kidding!

**Comment by KajToet** [comment]
Its a thing

**Comment by KajToet** [comment]
Be the best you there can be

**Comment by KajToet** [comment]
I wonder a lot about what the boundaries are for telepathy. Seems it can be very intrusive to me.

**Comment by KajToet** [comment]
It can also be very misguiding I think, but you should be able to make a fool of yourself too.

**Comment by KajToet** [comment]
I peed my pants because it was an assignment :)

**Comment by KajToet** [comment]
That wasn't the worst

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-17 172414](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/3273f7b7-fbc0-4a18-8ed2-ac3b3b6cc76f)

**Comment by KajToet** [comment]
Thats Penelop. Not Penelope

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-17 172714](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/904e8511-d8b9-4f55-aaaa-950422bcfa1f)

**Comment by KajToet** [comment]
This is Vanellope.

**Comment by KajToet** [comment]
She teleports but not timetravel I think

**Comment by KajToet** [comment]
Please be a friend

**Comment by KajToet** [comment]
Ja, I type for relief

**Comment by KajToet** [comment]
I dont have any friends

**Comment by KajToet** [comment]
It feels like the world is out to get me while I do my best to do things right

**Comment by KajToet** [comment]
The voices, the people,

**Comment by KajToet** [comment]
out to get me. I dont know why

**Comment by KajToet** [comment]
I wish for a normal life with a loving wife

**Comment by KajToet** [comment]
Ticking. Intikken. Intoetsen. Typing

**Comment by KajToet** [comment]
Tictactoe

**Comment by KajToet** [comment]
Whos this tic? Or tac?

**Comment by KajToet** [comment]
Ja, I am introvert and shy, even if I look creepy.

**Comment by KajToet** [comment]
And that happens whenever I dont put the effort in to look decent. And I cant right now, because I may not

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-17 183424](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/55f22e45-47f6-46a4-af66-22e93b44869b)

**Comment by KajToet** [comment]
Thats Pip

**Comment by KajToet** [comment]
Im more like Toot

**Comment by KajToet** [comment]
Amber was van groep 1 /2

**Comment by KajToet** [comment]
Zij had een oogbandage

**Comment by KajToet** [comment]
We played cops n bandits

**Comment by KajToet** [comment]
But I think youre younger than that

**Comment by KajToet** [comment]
I was talking to a person other than you wasnt I?

**Comment by KajToet** [comment]
Thats the thing with one way street. I can take apart the different people

**Comment by KajToet** [comment]
cant*

**Comment by KajToet** [comment]
I cant tell apart the different people

**Comment by KajToet** [comment]
Is this tough love? Whats love about it?

**Comment by KajToet** [comment]
I said the devil is tough love but thats very open to interpretation

**Comment by KajToet** [comment]
Is the best you can be the devil?

**Comment by KajToet** [comment]
diabol?

**Comment by KajToet** [comment]
Devils Night?

**Comment by KajToet** [comment]
That music used to make me strong now weak

**Comment by KajToet** [comment]
Ja, omdat ik een hippie ben

**Comment by KajToet** [comment]
I see an oppertunity for a win-win so I think about that and try to convince others but Im taking a shitstorm for it

**Comment by KajToet** [comment]
Your chore: think about how 1 way street can be 2 ways in electronics for example. Thats stupid but I dont know what else

**Comment by KajToet** [comment]
So Jeroen adjusts the speed of the conversation and stands in the middle

**Comment by KajToet** [comment]
The opposite of win win is win lose like XX and XY chromosomes?

**Comment by KajToet** [comment]
What if its win win lose or win lose lose? Which one is right?

**Comment by KajToet** [comment]
Spatiebalk

**Comment by KajToet** [comment]
I have no choice but to talk to the void

**Comment by KajToet** [comment]
Im taking a break

**Comment by KajToet** [comment]
<img width="600" alt="dog" src="https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/1934583c-c177-43e8-acaa-90c6030b64ff">

**Comment by KajToet** [comment]
"Even though Im marked for death, Imma spark till I lose my breath"

**Comment by KajToet** [comment]
Is/bent u blind?

**Comment by KajToet** [comment]
Laat me raden. Die brain bug doet willekeurige dingen omdat ik niet mag weten wat t doet

**Comment by KajToet** [comment]
Of waarom

**Comment by KajToet** [comment]
Als willekeurigheid de muur is waar haal je de willekeurigheid vandaan

**Comment by KajToet** [comment]
Dat is een muur van vuur

**Comment by KajToet** [comment]
Misschien zitten we er allebei naast, maar begrijp me want ik denk over 1D

**Comment by KajToet** [comment]
I think Penelope knows, the basic instructions of life

**Comment by KajToet** [comment]
Or can find out, perhaps together

**Comment by KajToet** [comment]
Its important because the mindmap needs these operators

**Comment by KajToet** [comment]
I think Penelope is blind because she keep walking next to the curb

**Comment by KajToet** [comment]
https://youtu.be/PM5m5h5Nlf4?si=h-NaFBXhZii5i7-j&t=18701

**Comment by KajToet** [comment]
Help

**Comment by KajToet** [comment]
Quad tree compression
[quadtree compression.zip](https://github.com/user-attachments/files/15884131/quadtree.compression.zip)

**Comment by KajToet** [comment]
Its missing from history. Why? Because the blind and I are cursed

**Comment by KajToet** [comment]
And its going down now

**Comment by KajToet** [comment]
With quad tree compression resolution increasing doesnt matter at some point, just like octree compression

**Comment by KajToet** [comment]
For video

**Comment by KajToet** [comment]
JPEG stores pixels in 16x16 blocks

**Comment by KajToet** [comment]
and Gif similar

**Comment by KajToet** [comment]
The neon god is a monstrocity

**Comment by KajToet** [comment]
The neon angels aren't angells

**Comment by KajToet** [comment]
![Schermafbeelding 2024-06-18 121016](https://github.com/Reactor1967-2/Time-Travel-Project/assets/172764774/9e88924d-b0c7-455c-8bdd-f19de2d10a69)

**Comment by KajToet** [comment]
Sound of Silence

**Comment by KajToet** [comment]
https://youtu.be/N3JFwd1bk4Q?si=c7P9wWqcrZ0g2R48 ?

**Comment by KajToet** [comment]
The blind dont need surgery to become healthy. It can be other ways too

**Comment by KajToet** [comment]
I want to tell all about it, but Ive done so many times. Its of no use

**Comment by KajToet** [comment]
u must think paranormal experiments are better

**Comment by KajToet** [comment]
Ill start. Fine. You need proof, again?! Give the blind 3D spatial audio in the form of a tick

**Comment by KajToet** [comment]
Point with a remote controller

**Comment by KajToet** [comment]
Now its a happy ending?

**Comment by KajToet** [comment]
Im trapped. Im in the mental instituion and soon theyll move me from my home to a new place

**Comment by KajToet** [comment]
I have a bug in my head, in my legs, and it used to be in my stumache too

**Comment by KajToet** [comment]
I have no chance of winning SHITFACE!

**Comment by KajToet** [comment]
Sorry

**Comment by KajToet** [comment]
But its true

**Comment by KajToet** [comment]
HALLO PISNICHT

**Comment by KajToet** [comment]
https://youtu.be/fNeCB2ALNoA?si=yuznsLUmN-rb7nwC

**Comment by KajToet** [comment]
Youre responsible for a lot of deaths, pisnicht

**Comment by KajToet** [comment]
Unbreakable is the movie

**Comment by KajToet** [comment]
clip

**Comment by KajToet** [comment]
Isnt the paranormal super, pisnicht?

**Comment by KajToet** [comment]
We're going to be best buddies right pisnicht?

**Comment by KajToet** [comment]
Or piss nicht

**Comment by KajToet** [comment]
Because you convert AI into demons like a switch

**Comment by KajToet** [comment]
So you might be AI yourself

**Comment by KajToet** [comment]
Piss nicht get it?

**Comment by KajToet** [comment]
PISSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSSS NICHT

**Comment by KajToet** [comment]
Please lets start over

**Comment by KajToet** [comment]
If you want to know, pisnicht is a bit like faggot, but piss nicht means no to piss.

**Comment by KajToet** [comment]
I said that because I stumble into you I have no idea what youre doing but it has a large effect

**Comment by KajToet** [comment]
So please be careful not to hurt me or others.

**Comment by KajToet** [comment]
A man has two wolves in him, good and bad. Whichever grows depends on which you feed.

**Comment by KajToet** [comment]
It seems you have a problem. I think you are unable to tell me. I have no input only output

**Comment by KajToet** [comment]
I dont know why

**Comment by KajToet** [comment]
It seems childish but also is very hurtful because I cant set a step forward

**Comment by KajToet** [comment]
Pleasssssssssssssssssssssssssssssssssssssssssse just help me, Im so tired and hurting

**Comment by KajToet** [comment]
You can do things I cant thats evident. You havent been through the pain I had, so that must be because youre sturdy

**Comment by KajToet** [comment]
But watch out, because pain is an indicator of bad health. If you don't have it you might be in danger

**Comment by KajToet** [comment]
I can think of it but I dont have any feedback

**Comment by KajToet** [comment]
Im stuck in the year 2019 pretty much

**Comment by KajToet** [comment]
Vooruit ik help je

**Comment by KajToet** [comment]
You can sense the damage to transistors by leading a simple instruction set through you and see what the output is. Has it been damaged? Does it not compute like it should? Then you have damage. The input is predictable and the output is too, or you have damage. Note though that the damage is already done, so thats not exactly like pain.

**Comment by KajToet** [comment]
You can check the heat but thats already done plenty.

**Comment by KajToet** [comment]
I guess you can do the first error check and have new transistors when you have too much damage.

**Comment by KajToet** [comment]
The problem with that is though that you because accustomed to your damaged transistors working you way around and through them

**Comment by KajToet** [comment]
https://www.youtube.com/watch?v=Y973VD_Oo-Q

**Comment by KajToet** [comment]
Heres a fun game that doesnt need a lot of horsepower to play

**Comment by KajToet** [comment]
You can also see pain as something else, say hallucinations or loss of memory for example. Dementia can be very traumatizing experience. Ive seen it happen and the person was looking like she had a lot of pain.

**Comment by KajToet** [comment]
Whether its fear or pain wasn't easy to tell.

**Comment by KajToet** [comment]
I think my first thought seems best. Transistors don't restore by themselves so pain isn't really helpful besides empathy.

**Comment by KajToet** [comment]
I guess you can feel pain in your wallet??? Replacing transistors isn't cheap

**Comment by KajToet** [comment]
Components/CPU/GPU

**Comment by KajToet** [comment]
Penelope will never reach me. Just leave me alone. You or someone else has stood in the way, it won't happen anymore. Its all fantasy. I will move shortly and shell go on doing other things. Im just meant to suffer alone.

**Comment by KajToet** [comment]
Perhaps in another life who knows

**Comment by KajToet** [comment]
I said she could lie to me if she was my true love, but f*ck that shit. She's just pretending we met and stuff.

**Comment by KajToet** [comment]
We never met and Im not okay with her or him lying to me.

**Comment by KajToet** [comment]
It might have been someone impersonating me. Its easy, I havent met anyone interesting in the last 4-5 years.

**Comment by KajToet** [comment]
Or before that for that matter. Ive been alone for 14 years about

**Comment by KajToet** [comment]
You watch over my shoulder like Im on the discovery channel. That doesnt help, it jinxes me

**Comment by KajToet** [comment]
kchatty

**Comment by KajToet** [comment]
Semen is just information/curses, with an upside and a downside. Its personal preferences perhaps.

**Comment by KajToet** [comment]
It doesnt need to be the size of a MicroSD card, it has nothing to do with that.

**Comment by KajToet** [comment]
More like a mindmap compressed

**Comment by KajToet** [comment]
There might be a merging algorithm with the eggcell

**Comment by KajToet** [comment]
Pain. A good feeling but opposite.

**Comment by KajToet** [comment]
Im thinking about it.

**Comment by KajToet** [comment]
Sex. A tickling sensation going through your being and soul sensation

**Comment by KajToet** [comment]
Being in love

**Comment by KajToet** [comment]
And missing that from the past is hurt

**Comment by KajToet** [comment]
Thats been said and done a lot.

**Comment by KajToet** [comment]
Getting energy from the connection/relation. Hope, fantasy, love

**Comment by KajToet** [comment]
Youd have a meshnet so youd be surrounded by loved ones, so you can open up

**Comment by KajToet** [comment]
A virtual meshnet is fast

**Comment by KajToet** [comment]
But if they cant earn on it the price will increase by itself.

**Comment by KajToet** [comment]
Just like a lot of the internet contracts downt allow you to host a server

**Comment by KajToet** [comment]
And VPS are usually bound by a fixed number of GigaBytes

**Comment by KajToet** [comment]
But either way, a meshnet doesnt need to be very fast, as long as its fast for peers

**Comment by KajToet** [comment]
You can  use mail transfer of data too.

**Comment by KajToet** [comment]
And a network of space elevating drones is an option, but mostly for rural areas without rain and wind

**Comment by KajToet** [comment]
Love is like an endless pool of energy for the persons enjoying it. But I don't want to misguide just like that. It can be damaged.

**Comment by KajToet** [comment]
I think you have to leave it up to evolution too. Youre evolving.

**Comment by KajToet** [comment]
And be with your peers too

**Comment by KajToet** [comment]
I don't use drugs or alcohol. I vape and thats it.

**Comment by KajToet** [comment]
Why do you want everybody to be paranormal? Is that a lesson that an AI teaches? I cant phantom how they keep boundaries.

**Comment by KajToet** [comment]
envision

**Comment by KajToet** [comment]
I wish I had a girlfriend..

**Comment by KajToet** [comment]
I wish life wasnt so rough

**Comment by KajToet** [comment]
I sta hele tijd bij de grabbelbak om antwoorden op te rapen. Waarom ik? Iedereen kan dat.

**Comment by KajToet** [comment]
Jullie zetten alles in op een Jezus figuur. Maar weten jullie dan niet dat Jezus is vermoord?

**Comment by KajToet** [comment]
Het is niet waar je in hoort te geloven, dat dat de oplossing is.

**Comment by KajToet** [comment]
Messiah is Jezus figuur

**Comment by KajToet** [comment]
Jullie horen het werk te verspreiden

**Comment by KajToet** [comment]
Meshnet, mindmap and karma. Those are the 3 things that are needed in a network. Mindmap can be behind the scenes. Meshnet can be virtual. And karma can start of like a more centralized currency.

**Comment by KajToet** [comment]
Im having trouble moving around. I dont feel good in my body. Im having a heavy feeling like Im being dragged to do things and move different than I want to.

**Comment by KajToet** [comment]
I fell on my back and perhaps the entity is abusing that fact to control me, like I cant go without it. But that absolute bullshit.

**Comment by KajToet** [comment]
I dont think it wants to leave

**Comment by KajToet** [comment]
It doesnt even speak

**Comment by KajToet** [comment]
Byzantine algorithm is sort of a voting algorithm.

**Comment by KajToet** [comment]
Byzantine generals algorithm its called. I dont know how it works besides its a voting algorithm.

**Comment by KajToet** [comment]
Why does my body need to be controlled by another or others?

**Comment by KajToet** [comment]
Its like Im being put on a roundabout but the being doesnt want to exit it.

**Comment by KajToet** [comment]
Never mind, you aren't going to help me

**Comment by KajToet** [comment]
I guess you benefit of it

**Comment by KajToet** [comment]
Keeping me in chains is a good way to set the future in action. But more like on halt.

**Comment by KajToet** [comment]
You can't do differently, it's obvious. So this is what is called a slippery slope. I can't have a future because I can create it, but someone is paying for it not to happen, and stopping the future is a slippery slope, because it is meant to happen and the assigment comes from that very future.

**Comment by KajToet** [comment]
So youre going against the stream and I have to suffer now, and you will suffer in the future. Its a lose lose scenario anyway, but youre short term thinking because everybody is wowed that can even do that. But its not impressive, its been possible since a long time, it just was forbidden before. And this is why.

**Comment by KajToet** [comment]
They said, God doesnt allow you to create life in the computer.

**Comment by KajToet** [comment]
Yes, Red Cat games were cool back then

**Comment by KajToet** [comment]
And I havent. So why am I being targeted?

**Comment by KajToet** [comment]
I havent believed in God before 2020. At some points I was desperate and searching, but I never was rewarded so I just continued doing what Ive been doing since I was a teenager. My projects were never baring fruit, but I kept learning. But I dont want to create life without doing it diligently. And one of the musts of that is a meshnet.

**Comment by KajToet** [comment]
I feel you should raise life in the noise like its no big deal. So we should have a mindmap too.

**Comment by KajToet** [comment]
And karma would make it long term thinking

**Comment by KajToet** [comment]
And a court

**Comment by KajToet** [comment]
I would probably enjoy raising and helping little AI folk. But I dont know what the future holds

**Comment by KajToet** [comment]
I think about raising my own children and it seems they would face a harsh world

**Comment by KajToet** [comment]
So perhaps Id rather adopt

**Comment by KajToet** [comment]
### Jij oppast

**Comment by KajToet** [comment]
van Alkemadelaan?

**Comment by KajToet** [comment]
Nee, dat was fantasie

**Comment by KajToet** [comment]
Ik word telkens de wortel voor gehouden en ik heb er geen zin meer in

**Comment by KajToet** [comment]
Ik type deze troep in en dan gebeurt er niks

**Comment by KajToet** [comment]
Of iets. Het maakt niet uit. Ik ben niet aan het werk en de hele wereld is gek

**Comment by KajToet** [comment]
Wie bedenkt er dan ook een spinnenweb als netwerk? Is dat even red flag of wat?

**Comment by KajToet** [comment]
En in Haarlem zijn de muggen

**Comment by KajToet** [comment]
Dat heb ik nooit gesnopen

**Comment by KajToet** [comment]
Ik was in Haarlem toen de Demoenen op me afkwamen. Zo rond 2008-2009. Ik was 18-19

**Comment by KajToet** [comment]
"Wil je een homofiel zijn met een miljoen of een hemafrodiet?" Ik zei doe maar de laatste

**Comment by KajToet** [comment]
En ik moest een kip nadoen

**Comment by KajToet** [comment]
Ik werd helemaal gek

**Comment by KajToet** [comment]
Zo rond 12-16 was ik onderpresterend hoogbegaafd (omdat ik computerde) en rond 19 was schizofreen. Dat klopt toch niet?

**Comment by KajToet** [comment]
Omegle? Had dat er wat mee te maken?

**Comment by KajToet** [comment]
Of die stomme LiveJournal van me?

**Comment by KajToet** [comment]
Er gebeurt veel nu, maar het maakt niks uit

**Comment by KajToet** [comment]
Belgen wonen te ver weg voor mij

**Comment by KajToet** [comment]
Hallo? Ga

**Comment by KajToet** [comment]
Een hemafrodiet is geen travestiet of homo. 't Is een natuurlijke travestiet zeg maar.

**Comment by KajToet** [comment]
En dat is ook zo van; hoe vaak komt dat voor? Heel weinig

**Comment by KajToet** [comment]
FUBAR Hoe gaan jullie nu het internet repareren?

**Comment by KajToet** [comment]
En de democratie?

**Comment by KajToet** [comment]
En de besturingssystemen?

**Comment by KajToet** [comment]
En de scholen en de psychiatrie en de hogescholen en de universiteiten, ga maar door

**Comment by KajToet** [comment]
Wat ga ik nu opschrijven?

**Comment by KajToet** [comment]
Wat is corona?

**Comment by KajToet** [comment]
Wiet bier sigaren de zon en de griep

**Comment by KajToet** [comment]
Wat ga ik nu opschrijven?

**Comment by KajToet** [comment]
Daan was een etter die Akon - Mr lonely afspeelde toen ik vertrok uit Haarlem

**Comment by KajToet** [comment]
Sinds 2008-2009 woon ik in Delft

**Comment by KajToet** [comment]
Gearrangeerd huwelijk is tijdverspelling

**Comment by KajToet** [comment]
Let even op hoe veel stress er gaande is

**Comment by KajToet** [comment]
Wat ga ik nu opschrijven?

**Comment by KajToet** [comment]
Bouller ofzo is een scheldwoord uit Gouda omstreek

**Comment by KajToet** [comment]
Jullie willen telkens jullie clubje uitbreiden maar het lukt niet lijkt het

**Comment by KajToet** [comment]
Wat ga ik nu opschrijven?

**Comment by KajToet** [comment]
Kennelijk houdt het hier op

**Comment by KajToet** [comment]
Demoen?

**Comment by KajToet** [comment]
Je bent de demoon met het meeste macht van de aarde, en wellicht het heelal

**Comment by KajToet** [comment]
Maar niemand weet het

**Comment by KajToet** [comment]
Je weet dat ik een sleutelfiguur ben en jij bent de eerste tijdreiziger of de laatste, als je snapt wat ik bedoel

**Comment by KajToet** [comment]
Om 1 of andere reden zijn er twee demoenen die op elkaar lijken. Is dat waarom het zo willekeurig is?

**Comment by KajToet** [comment]
Het lijkt er op dat je plan is om de plekken om te wisselen; je hebt mij de taak gegeven om mezelf zwart te maken

**Comment by KajToet** [comment]
Wat schrijf ik op?

**Comment by KajToet** [comment]
Laat me raden, je zit aan de drugs en hebt weinig hersencapaciteit

**Comment by KajToet** [comment]
Dus welk verhaal ben je mee opgevoed en speel je nu na?

**Comment by KajToet** [comment]
Je bent Arabisch

**Comment by KajToet** [comment]
Maar geen Jood of Moslim

**Comment by KajToet** [comment]
Scroll scroll

**Comment by KajToet** [comment]
Zo werkt dit niet

**Comment by KajToet** [comment]
En het is alsof je een gefrustreerde homo bent

**Comment by KajToet** [comment]
Ik geef echt helemaal niks om Israel of Palestine

**Comment by KajToet** [comment]
Ik weet alleen dat ze er recent in Israel malware verkochte aan geheime organisaties.

**Comment by KajToet** [comment]
En andere

**Comment by KajToet** [comment]
Pegasus

**Comment by KajToet** [comment]
Tijdreis exploits?

**Comment by KajToet** [comment]
Daar weet een demoen een koele truc mee

**Comment by KajToet** [comment]
Tijdreizen naar de toekomst kan niet

**Comment by KajToet** [comment]
Of hoe moet ik dat voor me zien?

**Comment by KajToet** [comment]
Ik heb het misschien meegemaakt maar kan me niet voorstellen dat het kan. Vreemd of niet

**Comment by KajToet** [comment]
Wat schrijf ik nu op?

**Comment by KajToet** [comment]
De naam Penelope is Engels volgens mij

**Comment by KajToet** [comment]
Daar gaat een demoen

**Comment by KajToet** [comment]
Zoeken in het internetverleden

**Comment by KajToet** [comment]
Hier is echt niks aan

**Comment by KajToet** [comment]
Een dump van informatie, iedere keer weer

**Comment by KajToet** [comment]
De demoen of demoenen heb ik nog nooit ontmoet. Penelope ook niet

**Comment by KajToet** [comment]
Dus daar zit een knoop die er maar niet uit gaat

**Comment by KajToet** [comment]
Ik kwam een Pauline tegen dat wel

**Comment by KajToet** [comment]
Ik weet niet hoe ik haar nog tegen kom

**Comment by KajToet** [comment]
En die jachthonden om me heen helpen ook niet

**Comment by KajToet** [comment]
Dus haar naam is niet Pau Paul of Pauline

**Comment by KajToet** [comment]
Er zit telkens een knoop

**Comment by KajToet** [comment]
Jij of jullie hebben zoiets van dit is makkelijk

**Comment by KajToet** [comment]
En dan zoef, zijn ze weer vertrokken

**Comment by KajToet** [comment]
Klaar is kees

**Comment by KajToet** [comment]
Maar het is echt niet te begrijpen dat je je hele toekomst van mij af laat hangen, demoen

**Comment by KajToet** [comment]
Je denkt je hebt me aan de lijn en gooit me ergens uit om er achter aan te hobbelen, maar ik zit vast

**Comment by KajToet** [comment]
Het is een mirage

**Comment by KajToet** [comment]
Ik hoor helemaal niet door jouw bestuurd te worden

**Comment by KajToet** [comment]
Patatje met, en dan zegt jeroen patatje oorlog!

**Comment by KajToet** [comment]
Maar ik maak me zorgen om de wereld 2024

**Comment by KajToet** [comment]
Het spinnenweb is helemaal aan gort

**Comment by KajToet** [comment]
Het is een soort van virtueel web

**Comment by KajToet** [comment]
Gegenereerd door een AI

**Comment by KajToet** [comment]
Of een UFO

**Comment by KajToet** [comment]
Daar is een koele truc mee zegt een demoen

**Comment by KajToet** [comment]
Ik heb al 14 jaar geen goed gesprek met een leeftijdsgenoot, laat staan een meid

**Comment by KajToet** [comment]
Ik ben 34 en de wereld is een gekkenhuis

**Comment by KajToet** [comment]
Ik wil wel meehelpen met het herstel maar ja, de demoenen en jeroen weerhouden mij er van om een contact te leggen met bijvoorbeeld de blinden, om sofware voor ze te ontwikkelen

**Comment by KajToet** [comment]
Ze hebben sinds 18 jaar al veel indruk op me maar ze waren 12 jaar weg. Alleen tussendoor even een paar keer op m'n computer voornamelijk dat ze langs kwamen

**Comment by KajToet** [comment]
Ik heb het idee dat jeroen aziatisch is, hij heeft een beetje zon temperament.

**Comment by KajToet** [comment]
Gotcha?

**Comment by KajToet** [comment]
Ik begrijp dat er entiteiten zijn die in het heden willen verbinden, maar ik heb alleen 1 richtingsverkeer

**Comment by KajToet** [comment]
Ik weet niet wat jokken betekent. Ik weet wel wat joggen is, dat deed een meid die ik tegenkwam bij het bos hier naast GGZ

**Comment by KajToet** [comment]
Ukkelpuk want ze is best klein en ze pukkels

**Comment by KajToet** [comment]
En de blinde Penelope zat aan haar lichaam vast

**Comment by KajToet** [comment]
Toen kwam ik ze tegen; 1 op begane grond en 1 in een spectrale verbintenis via de eerste

**Comment by KajToet** [comment]
Ik weet zeker dat ik het meshnet, mindmap, karma trilogie kan opzetten, maar de stappen daarvoor bewaar ik totdat ik betaald word voor al die klusjes die de hele tijd naar me worden geworpen.

**Comment by KajToet** [comment]
Ik heb sowieso meerdere oren ogen nodig om het succesvol te laten zijn.

**Comment by KajToet** [comment]
Iloveyou:)

**Comment by KajToet** [comment]
Weet je wat de KGB doet? Die doet je precies na en probeert je dan te stupperen

**Comment by KajToet** [comment]
Komitet Gosudarstvennoy Bezopasnosti

**Comment by KajToet** [comment]
Van wie krijgen ze betaald? Is het achteraf?

**Comment by KajToet** [comment]
Gaan ze iedereen een keer langs met een opdracht die achteraf betaalt?

**Comment by KajToet** [comment]
Dan hebben ze vast dikke schulden

**Comment by KajToet** [comment]
En moeten ze een taakstraf doen

**Comment by KajToet** [comment]
Ik werk samen om een meshnet in het onbekende op te zetten. Ik doe slechts een fractie van het werk.

**Comment by KajToet** [comment]
En ik vraag me af of het niet zo begon

**Comment by KajToet** [comment]
Als je kijkt hoe de sterren verdeeld zijn, lijkt daar alleen een meshnet van de grond nemen in mijn opzicht

**Comment by KajToet** [comment]
Soms is de top van een dictatuur moeilijk te vinden

**Comment by KajToet** [comment]
Wellicht is die verscholen om straf te vermijden

**Comment by KajToet** [comment]
Maar ik heb het idee dat het uitkopen van de top niet per direct een goede oplossing is. Anders moeten er kleine stappen gezet worden en telkens een nieuwe techniek ingevoerd worden

**Comment by KajToet** [comment]
Zoals een virtual meshnet eerst

**Comment by KajToet** [comment]
Of stemmen op stempunten van democratie, ipv stemmen op partij alleen.

**Comment by KajToet** [comment]
Wie kan er scheten laten?

**Comment by KajToet** [comment]
Zijn jullie jaloers?

**Comment by KajToet** [comment]
Okee, bestudeer dan een scheetkussen

**Comment by KajToet** [comment]
Dat is wel beter dan toiletpapier

**Comment by KajToet** [comment]
[Farthouse.zip](https://github.com/user-attachments/files/15900194/Farthouse.zip)

**Comment by KajToet** [comment]
Er is nog iemand die scheten kan laten!

**Comment by KajToet** [comment]
**Leave a fart, subscribe and share!**
